app/etc/modules/Kirchbergerknorr_UrlCacheControl.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * @package     Kirchbergerknorr\CoreFixes
```

Comment:
This package is a copy+paste leftover I guess.

Reply:
Yes, will remove it